packages/manager/apps/telecom/src/app/telecom/telephony/alias/administration/terminate/translations/Messages_fr_FR.json

```diff
@@ -22,5 +22,11 @@
   "telephony_alias_administration_terminate": "Résilier",
   "telephony_alias_administration_terminate_success": "Résiliation planifiée avec succès.",
   "telephony_alias_administration_cancel_termination": "Annuler la résiliation",
-  "telephony_alias_administration_cancel_termination_success": "Résiliation annulée."
-}
\ No newline at end of file
+  "telephony_alias_administration_cancel_termination_success": "Résiliation annulée.",
+  "telephony_alias_administration_terminate_sva_warning": "Une résiliation étant en cours, les versements seront encore possible pendant les 2 prochains mois (60 jours).<br>Afin de garantir ses versements il est important de maintenir sur cette période votre KYC valide ainsi que le compte de versement existant.",
```

Comment:
**question:** Couldn't be splitted into multiple translation instead?

Reply:
Yes we could, we just receive content and we organize it the way we want. Does it change something to split it?

Reply:
As a translator, is more convenient to not having to deal with HTML tags.
Since you're only using a `<br>` element, then it could be splitted into two chunk.